### PR TITLE
chore: HttpLoadBalancerExtension should expose targetGroup

### PR DESCRIPTION
**Pull request by homakk**
Issue Related #209
Here the customer provided use case is not necessarily passing in a target group as a property in the LB constructor, but to expose the target group as an optional field on the Service after it's created. 

So when you add an HTTPLoadBalancerExtension, you can customize the health checks by accessing the targetGroup field on the service.

Here I'm providing an example how to access the targetGroup

```
  const service = new Service(stack, 'my-service', {
  environment,
  serviceDescription,
  autoScaleTaskCount: {
    maxTaskCount: 5,
  },
});

service.targetGroup.configureHealthCheck({
  path: '/',
  port: '80',
});
```

And Here I'm providing yarn compile and yarn test results as shown below:
image

### CODE REVIEW

File: src/service.ts [hunk 1]
```diff
@@ -128,6 +129,11 @@ export class Service extends Construct {
    */
   public readonly scalableTaskCount?: ecs.ScalableTaskCount;
 
+  /**
+   * The application target group.
```
Comment (bvtujo) [inline src/service.ts:133]: Please elaborate. 

```suggestion
   * The application target group, if the service has an HTTPLoadBalancerExtension.
```

File: src/extensions/http-load-balancer.ts [hunk 2]
```diff
@@ -72,7 +72,7 @@ export class HttpLoadBalancerExtension extends ServiceExtension {
       }
       this.parentService.scalableTaskCount.scaleOnRequestCount(`${this.parentService.id}-target-request-count-${this.requestsPerTarget}`, {
         requestsPerTarget: this.requestsPerTarget,
-        targetGroup,
+        targetGroup: this.parentService.targetGroup || targetGroup,
```
Comment (bvtujo) [inline src/extensions/http-load-balancer.ts:75]: this will always result in this line using `targetGroup`, not `this.parentService.targetGroup`.

parentService.targetGroup isn't set. Please add a line somewhere which sets the target group of the parentService so that customers can actually use it.

File: src/service.ts [hunk 3]
```diff
@@ -128,6 +129,11 @@ export class Service extends Construct {
...
+   */
+  public targetGroup?: ApplicationTargetGroup;
```
Comment (bvtujo) [inline src/service.ts:135]: We need to test that this is set (ie non-null) after adding an HTTPLoadBalancer extension.

**Review by bvtujo — COMMENTED** [review]
looks good. Before we merge this, though, can you update the documentation with an example of how a customer might want to use the target group? You can look at the linked issue to see what the original customer was asking about.

**Comment by bvtujo** [comment]
Can you add this example you've provided to the readme?

**Review by bvtujo — CHANGES_REQUESTED** [review]

**Review by bvtujo — APPROVED** [review]
